fixtures/webversions.py

```diff
@@ -0,0 +1,48 @@
+from dotenv import load_dotenv
+import os
+import pytest
+import urllib.request
+from ast import literal_eval
+load_dotenv(verbose = True)
+url_addr_version = os.environ['URL_ADDR_VERSION']
+
+# adding cnx-deploy version number to applitools screenshot results
+@pytest.fixture(scope="session")
+def deploy_version_tag(request):
+    """Returns cnx version numbers"""
+
+    version_page = urllib.request.urlopen(url_addr_version)
```

Comment:
`version_page = urllib.request.urlopen(url_addr_version)`

We should try to avoid globals. What's another way for you to pass `url_addr_version` into this function?